architecture/platforms/versal_acap.rst

```diff
@@ -0,0 +1,125 @@
+.. _versal_acap:
+
+#############################################
+VERSAL Adaptive Compute Acceleration Platform
```

Comment:
List all of the related configuration flags for your port and what they do.

Reply:
List how customers can build your version of OP-TEE in case they want to build without a factory.

Reply:
No, the flags are described in the code.

Reply:
That's not documentation. That's code. We'll need these pulled out in a single document so that customers don't have to go digging around the code to first find them and then figure out what they do.

Reply:
The definition of documentation depend on the project, the industry, culture: have you read the OP-TEE platform documentation to have an informed point of view? I guess not. Check the documentation and you will see that not all platform specific flags are documented (in fact, very few if any are)
It seems to me you are asking for a detailed document you can present to users with little OP-TEE experience and little exposure to Versal ACAP.

Reply:
Hi Jorge,
Yes - I read through the OP-TEE documentation and GP docs at least once a year to keep up on any major changes. And I agree - documentation of flags in general is not really there. And you are correct, we'll be sending customers to you that have little to no OP-TEE experience and this could save you the trouble of customer support in the future. But all in all, however you want to document is up to you as I'm just providing my feedback and suggestions. With regards to Versal ACAP security they can first read the TRM, etc., talk to their sales rep, and then come to the Security CoE so AMD-Xilinx will have that handled.

Reply:
As a general comment, document flags in the documentation is appreciated although we haven't been pushing for this nor being the most active ones ourselves. But, a page where we've done some flag documentation is the [TA properties](https://optee.readthedocs.io/en/latest/architecture/trusted_applications.html). If we can keep code and documentation in sync, then I'd be happy to see more flags documented in the documentation.